shared/tasks/browser.mjs

```diff
@@ -30,7 +30,21 @@ export async function screenshots () {
 
   // Screenshot components
   for (const componentName of componentNames) {
+    const componentExamples = await getExamples(componentName)
+
+    // Check for optional "inverse" example
+    const exampleInverted = Object.entries(componentExamples)
+      .find(([exampleName, data]) => typeof data.classes === 'string' && data.classes.endsWith('--inverse'))
```

Comment:
I'm a little antsy about this check being what determines whether Percy considers it inverse or not. 

Given the `classes` parameter is a generic one, there is no guarantee that `--inverse` be the last part of the string—there could be other classes after it. Maybe we have something like `govuk-button--inverse govuk-button--small` at some point? 

Not going to block on that basis alone, as it works for our current needs, but doing something more future-proof now might save us some confusion/debugging/refactoring down the line.

Reply:
Yeah you're spot on, that extra class example is a good one 😣

Have a look now? I've edited the component data so we always have both:

```yaml
- name: default
- name: inverse
```

Can just match on the example name `inverse` now

Reply:
Yeah, that seems a better approach to me.

This has made me wonder why until now we've only done the 'default' versions of components. There seems to be a few places where having Percy keeping track of prominent variants would be useful (the two different designs of the Pagination component, an open Details component, start Buttons, etc.) but that's outside of the immediate scope of this PR. We can add those in later if we think it's valuable.

Reply:
Thanks @querkmachine

They're really quick to add to moved your suggestions to:

* https://github.com/alphagov/govuk-frontend/pull/3962